bin/cylc-get-host-metric

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2018 NIWA
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""cylc get-host-metric [OPTIONS]
+
+   Get metrics for localhost, in the form of a JSON structure with
+   top-level keys as requested via the OPTIONS:
+
+   1. --load
+      1, 5 and 15 minute load averages (as keys) from the 'uptime' command.
+   2. --memory
+      Total free RAM memory, in kilobytes, from the 'free -k' command.
+   3. --disk-space=PATH / --disk-space=PATH1,PATH2,PATH3 (etc)
+      Available disk space from the 'df -Pk' command, in kilobytes, for one
+      or more valid PATHs within the filesystem of localhost. Multiple PATH
+      options can be specified via a comma-delimited list, each becoming a
+      key under the top-level disk space key.
+
+   If no options are specified, --load and --memory are invoked by default.
+
+   Note: designed as a light-weight command for localhost; to run on a remote
+         host, run after accessing the remote via existing functionality."""
+
```

Comment:
(I don't really understand what the "Note:..." means - can you explain?)

Reply:
Sorry if this isn't clear. I wanted to make it explicit that the command in itself was designed (with capability) only to run on localhost, & if someone wanted to run on a remote host, they would have to e.g. run via ``remote_cylc_cmd()`` or ssh to that host before running the command. In other words that a ``--host`` option or similar was not supported (does that make sense now?)

Re-reading this seems self-evident given the enumerated options which can be taken to be exhaustive. I will remove it.